### PR TITLE
Add BinanceTestNet Configuration

### PR DESCRIPTION
**Pull request by aadorian**
Add Binance TestNet Configuration and dotenv
Screen Shot 2020-12-26 at 11 17 59 PM
Screen Shot 2020-12-26 at 11 14 40 PM

### CODE REVIEW

File: scripts/fund-contract.js [hunk 1]
```diff
@@ -7,16 +7,16 @@ const LinkTokenInterface = artifacts.require('LinkTokenInterface')
...
-const payment = process.env.TRUFFLE_CL_BOX_PAYMENT || '1000000000000000000'
+//const payment = process.env.TRUFFLE_CL_BOX_PAYMENT || '1000000000000000000'
+const payment = process.env.TRUFFLE_CL_BOX_PAYMENT || '10'
 
 module.exports = async callback => {
   try {
-    const mc = await MyContract.deployed()
-    const tokenAddress = await mc.getChainlinkToken()
+    const mc = '0x162f637509672A6bD993D63c1a21F8d310971505' // your address
+    const tokenAddress = '0x761429d8887600dacE56BfA13651C018C1F0C538' //address of the deploy MyContract
```
Comment (thodges-gh) [inline scripts/fund-contract.js:16]: Having to change a deployment script's source is not user friendly. Can we make these environment variables instead that also take into consideration the deployed contract?

File: scripts/fund-contract.js [hunk 2]
```diff
@@ -7,16 +7,16 @@ const LinkTokenInterface = artifacts.require('LinkTokenInterface')
   contract for ease-of-use. Any extra LINK present on the contract
   can be retrieved by calling the withdrawLink() function.
 */
-
-const payment = process.env.TRUFFLE_CL_BOX_PAYMENT || '1000000000000000000'
+//const payment = process.env.TRUFFLE_CL_BOX_PAYMENT || '1000000000000000000'
```
Comment (thodges-gh) [inline scripts/fund-contract.js:10]: Please don't comment out code.

**Review by thodges-gh — CHANGES_REQUESTED** [review]

**Comment by se3000** [comment]
@aadorian it looks like you resolved the comments but didn't push the fixes. Mind pushing those?

**Comment by aadorian** [comment]
> @aadorian it looks like you resolved the comments but didn't push the fixes. Mind pushing those?

Shure I will close this and pr ...

**Pull request closed without merging**